### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by eltontay**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1,4 +1,18 @@
 [
+     {
+    "ticker": "WOT",
+    "poolId": "e00dac4566812442174bcfb4c12225f7a305606eed12f62e62085253",
+    "social": {
+      "website": "https://www.wottlenft.com",
+      "twitter": "WottleNFT Pool",
```
Comment (Bez625) [inline registry.json:7]: I cannot find this twitter account

File: registry.json [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+      "facebook": " ",
+      "instagram": "wottlenft",
+      "youtube": "wottlenft"
+    },
+    "telegram": "",
+    "operatorName": "Kopy-O",
```
Comment (Bez625) [inline registry.json:13]: This must be your real name or leave blank

File: registry.json [hunk 3]
```diff
@@ -1,4 +1,18 @@
...
+    "ticker": "WOT",
+    "poolId": "e00dac4566812442174bcfb4c12225f7a305606eed12f62e62085253",
+    "social": {
+      "website": "https://www.wottlenft.com",
+      "twitter": "WottleNFT Pool",
+      "facebook": " ",
+      "instagram": "wottlenft",
+      "youtube": "wottlenft"
+    },
+    "telegram": "",
```
Comment (Bez625) [inline registry.json:12]: You need to add the TG of an individual - not a group or channel

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 3 above)
Comment (eltontay) [inline registry.json:12]:
```suggestion
    "telegram": "nftay",
```

File: registry.json (hunk 1 above)
Comment (eltontay) [inline registry.json:7]:
```suggestion
      "twitter": "wottlenft",
```

File: registry.json (hunk 2 above)
Comment (eltontay) [inline registry.json:13]:
```suggestion
    "operatorName": "Elton Tay",
```

File: registry.json (hunk 1 above)
Comment (eltontay) [inline registry.json:7]:
```suggestion
      "twitter": "wottlenft",
```

**Pull request merged**